### PR TITLE
Adding ability to download VTEC's by state

### PR DESCRIPTION
**Pull request by mbalonso**
Adding ability to download VTEC's by state (#47)

### CODE REVIEW

**Comment by akrherz** [comment]
Thanks for this, I am out of the office for the next few days, so it will take me a bit to review it.

**Comment by mbalonso** [comment]
No problem.  Safe travels

File: include/forms.php [hunk 1]
```diff
@@ -2,12 +2,11 @@
 /**
  * Library for doing repetetive forms stuff
  */
-function make_select($name, $selected, $ar, $jscallback="", $cssclass=''){
```
Comment (akrherz) [inline include/forms.php]: This change breaks other parts of the website. Can you describe the need for it?

Comment (mbalonso) [inline include/forms.php]: Ah crap, I think that was accidental. Maybe the extra argument was added after the changes I made? Maybe I had a stale branch.

Comment (mbalonso) [inline include/forms.php]: I think the only intended changes were the new args in stateSelect.

**Comment by akrherz** [comment]
Thanks @mbalonso , it is merged and on the website, does it work there? :)

**Comment by mbalonso** [comment]
Thanks @akrherz.  Hmm, for some reason it is returning a different name for the downloadable file.  In-laws are staying w/us this weekend but I will troubleshoot.  Sorry :(

**Pull request merged**